.changelog/13304.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+telemetry: Added a `is_leader` metric to track if a server is a leader or not.
```

Comment:
let's use the full name:

```suggestion
telemetry: Added a `consul.server.is_leader` metric to track if a server is a leader or not.
```

Reply:
Also, I'm sorry I didn't think about it before, but we probably don't want to use `_` in the name as `_` can be a special character (for instance, for prometheus metrics). We have some metrics today that have `_` in their name but I think they are in the minority.

https://www.consul.io/docs/agent/telemetry

Let's do Carmel case: `isLeader` -- what do you think?

Reply:
Sure, I can make the change along with the other changes.